vmm/src/device_manager.rs

```diff
@@ -2079,17 +2492,24 @@ impl DeviceManager {
             .push(Arc::clone(&mmio_device_arc) as Arc<Mutex<dyn BusDevice>>);
         self.address_manager
             .mmio_bus
-            .insert(mmio_device_arc.clone(), mmio_base.0, MMIO_LEN)
+            .insert(mmio_device_arc.clone(), mmio_base, MMIO_LEN)
             .map_err(DeviceManagerError::BusError)?;
 
         self.cmdline_additions.push(format!(
             "virtio_mmio.device={}K@0x{:08x}:{}",
-            MMIO_LEN / 1024,
-            mmio_base.0,
+            mmio_size / 1024,
```

Comment:
Are we absolutely sure this is a multiple of 1KiB now?

Reply:
Yes we are since everything (store/restore of resources) is done by this function.